tests/data/snake_oil_data.py

```diff
@@ -12,9 +12,20 @@ def content(self):
         return self._content
 
 
+ENSEMBLE_NAME_3 = "default3"
+INDEX_RESPONSE = "FGPT"
+DATE_RESPONSE = "FOPR"
```

Comment:
I think if we have constants defined in a file they should be first after the import statements.

Reply:
thank you, fixed! surprising that neither black nor pylint pick up on that... apparently the python style guide doesn't have an opinion about it. huh